projects/core/src/process/store/selectors/process.selectors.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { TestBed } from '@angular/core/testing';
+import { select, Store, StoreModule } from '@ngrx/store';
+import * as fromActions from '../../../state/utils/entity-loader/entity-loader.action';
+import { LoaderState } from '../../../state/utils/loader/loader-state';
+import { PROCESS_FEATURE, StateWithProcess } from '../process-state';
+import * as fromReducers from '../reducers/index';
+import * as fromSelectors from './process.selectors';
+
+const MOCK_UPDATE_STATE = 'update-mock';
```

Comment:
`const MOCK_UPDATE_STATE = 'update-mock';`

Is that a fake process name?  It could be `MOCK_PROCESS_NAME =  'mock-process-name'` or something like that.  It would improve the code readability.

Reply:
Or MOCK_PROCESS_ID